common/testing/historyrequire/history_require.go

```diff
@@ -76,7 +76,9 @@ func (h HistoryRequire) EqualHistoryEvents(expectedHistory string, actualHistory
 
 	expectedHistoryEvents, expectedEventsAttributes := h.parseHistory(expectedHistory)
 
-	require.Equalf(h.t, len(expectedHistoryEvents), len(actualHistoryEvents), "Length of expected(%d) and actual(%d) histories is not equal", len(expectedHistoryEvents), len(actualHistoryEvents))
+	require.Equalf(h.t, len(expectedHistoryEvents), len(actualHistoryEvents),
+		"Length of expected(%d) and actual(%d) histories is not equal: \n%v",
+		len(expectedHistoryEvents), len(actualHistoryEvents), h.formatHistoryEvents(actualHistoryEvents, true))
```

Comment:
for the purpose of this PR it is fine.
But it can be more useful if we print diff.